tests/topotests/isis_topo1/test_isis_topo1.py

```diff
@@ -520,6 +520,28 @@ def test_isis_overload_on_startup_override_timer():
     check_lsp_overload_bit("r3", "r3.00-00", "0/0/1")
 
 
+def test_config_load_non_integrated():
+    tgen = get_topogen()
+    r3 = tgen.gears["r3"]
+    r3.vtysh_cmd(
+        f"""
+          configure
+            router isis 1
+              set-overload-bit
+        """
+    )
+
+    # Restart r3
+    stop_router(tgen, "r3")
+    start_router(tgen, "r3")
+
+    # Check that the overload bit is set in r3's LSP (without retry)
+    isis_database_output = r3.vtysh_cmd("show isis database r3.00-00 json")
```

Comment:
This sounds like "gonna fail sometimes", doesn't it? I mean checking without any retries or is this always a deterministic output after stopping/starting the router?

Reply:
Changed it to check the overload bit with a shorter timeout instead of eliminating retries